.gitignore

```diff
@@ -4,4 +4,6 @@ data/
 .idea/
 build
 .gradle
-run/
\ No newline at end of file
+run/
+# TODO REMOVE ME
```

Comment:
Ummm.....

Reply:
That was to protect against accidentally posting my bintray API key

Reply:
But it apparently didn't do anything lol